src/components/FormFeedback.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+
+class FormFeedback extends Component {
```

Comment:
Out of the scope of this PR, but maybe we think about what a more agnostic Form component would look like. We could build it in a way that is know how to consume a standard json schema. ex: https://github.com/mozilla-services/react-jsonschema-form